Tools/autotest/arducopter.py

```diff
@@ -5148,6 +5148,31 @@ def test_parameter_validation(self):
         self.drain_mav()
         self.assert_prearm_failure("Check MOT_PWM_MIN/MAX")
 
+    def test_alt_estimate_prearm(self):
+        self.context_push()
+        ex = None
+        try:
+            self.set_parameter("EK2_ALT_SOURCE", 2)
+            self.wait_gps_disable()
+            self.wait_heartbeat()
+            self.change_mode("ALT_HOLD")
+            self.assert_prearm_failure("Need Alt Estimate")
+            self.change_mode("STABILIZE")
+            self.zero_throttle()
```

Comment:
Shouldn't need to zero throttle rc input.  If you do the test suite has stuffed up somehow - it's one of the guarantees we make.

Reply:
done